lib/api.js

```diff
@@ -47,7 +48,7 @@ exports.default = function (_ref) {
   if (!module.exports) {
     // babel did not transform modules
     // eslint-disable-next-line no-underscore-dangle
-    component = module.__esModule ? module.default : module;
+    component = ctx.__esModule ? ctx.default : ctx;
```

Comment:
I guess it's not supported because there's no way to determine what's a Vue object or something completely unrelated.